notices: Add EigenDA v2 (#1977)
celo-org/docs: docs/cel2/notices/eigenda-v2-upgrade.md

# EigenDA v2 Upgrade

This page outlines changes related to the EigenDA v2 upgrade for node operators.

:::info

This page will be kept updated with key information about the upgrade. As this upgrade is activated on the sequencer, no detailed activation times can be given.

- Baklava testnet activation is planned for Wed, Jul 30, 2025.

> - Alfajores testnet activation is tentative planned for Wed, Aug 20, 2025.

Reviewer: ```suggestion
- Alfajores testnet activation is tentatively planned for Wed, Aug 20, 2025.
```

> - **Mainnet** activation is tentative planned for Wed, Sep 3, 2025.

Reviewer: ```suggestion
- **Mainnet** activation is tentatively planned for Wed, Sep 3, 2025.
```

:::

> ## What is the EigenDA v2 upgrade about?

Reviewer: I would add:
- link to Eigen's doc about it
- link to the forum post where we announced it (Eclair?)

> As part of Celo’s continued growth as an Ethereum L2, the Celo is preparing to integrate [EigenDA v2](https://docs.eigencloud.xyz/products/eigenda/core-concepts/overview), also known as Blazar, to further innovate and strengthen the network’s data availability layer.

Reviewer: ```suggestion
As part of Celo’s continued growth as an Ethereum L2, Celo is integrating [EigenDA v2](https://docs.eigencloud.xyz/products/eigenda/core-concepts/overview), also known as Blazar, to further innovate and strengthen the network’s data availability layer.
```

Blazar represents a major architectural upgrade to the EigenDA protocol, introducing improved system throughput and stability, alongside new capabilities like permissionless DA payments and enhanced resource throttling.

Most notably for Celo:

- End-to-end confirmation latency is significantly reduced, moving from minutes to near real-time. Blazar’s design enables rollups to reference blocks in their own logic without waiting for L1 confirmations.
- System throughput and network stability are greatly improved through more efficient chunk distribution, optimized request routing, and horizontal scalability of DA nodes.
Support for decentralized dispersal is unlocked by eliminating DDoS attack surfaces inherent in the original push-based mode.

## For node operators

> Node operators need to upgrade the [EigenDA proxy](https://github.com/Layr-Labs/eigenda-proxy) to version [v1.8.2](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.8.2) before the activation dates.

Reviewer: Maybe add that it is already safe to do so today
